Objective-C/TOCropViewController/TOCropViewController.m

```diff
@@ -84,7 +84,11 @@ - (instancetype)initWithCroppingStyle:(TOCropViewCroppingStyle)style image:(UIIm
         // Set up base view controller behaviour
         self.modalTransitionStyle = UIModalTransitionStyleCrossDissolve;
         self.modalPresentationStyle = UIModalPresentationFullScreen;
-        self.automaticallyAdjustsScrollViewInsets = NO;
+		if (@available(ios 11.0,	 *)) {
+			[UITableView appearance].contentInsetAdjustmentBehavior = NO;
```

Comment:
This would end up applying that appearance modifier to every instance of `UITableView` in the app, which probably isn't what we want here.